src/components/Bookinput.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Bookinput() {
+  return (
+    <form>
+      <input type="text" placeholder="book title" />
+      <button type="button">submit</button>
+    </form>
```

Comment:
- [ ] The CreateNewBook should render a form with inputs for the title and the author of the book. Please implement it.